### ISSUE TITLE
Use italic for spinbox block and setter height like other spinbox

### ISSUE THREAD

**Issue by alkabal** [linuxcnc-probe-screen/probe-screen-ng#31]
https://github.com/linuxcnc-probe-screen/probe-screen-ng/blob/f4e4db60f521f4b7baeccb68b7b9650a9b322fea/psng/python/psng.py#L177

```
    def on_spbtn_block_height_key_press_event(self, gtkspinbutton, data=None):
        keyname = gtk.gdk.keyval_name(data.keyval)
        #        print "Key %s (%d) was pressed" % (keyname, data.keyval)
        if keyname == "Return":
            gtkspinbutton.modify_font(pango.FontDescription("normal"))
        else:
            gtkspinbutton.modify_font(pango.FontDescription("italic"))
            
    def on_spbtn_probe_height_key_press_event(self, gtkspinbutton, data=None):
        keyname = gtk.gdk.keyval_name(data.keyval)
        #        print "Key %s (%d) was pressed" % (keyname, data.keyval)
        if keyname == "Return":
            gtkspinbutton.modify_font(pango.FontDescription("normal"))
        else:
            gtkspinbutton.modify_font(pango.FontDescription("italic"))
```
add : 

`<signal name="key-press-event" handler="on_spbtn_probe_height_key_press_event" swapped="no"/>`

https://github.com/linuxcnc-probe-screen/probe-screen-ng/blob/f4e4db60f521f4b7baeccb68b7b9650a9b322fea/psng/psng.glade#L2485

`<signal name="key-press-event" handler="on_spbtn_block_height_key_press_event" swapped="no"/>`

https://github.com/linuxcnc-probe-screen/probe-screen-ng/blob/f4e4db60f521f4b7baeccb68b7b9650a9b322fea/psng/psng.glade#L2508

**Comment by alkabal** [comment]
need also to add in "def on_spbtn_block_height_value_changed(self, gtkspinbutton, data=None):"
`        gtkspinbutton.modify_font(pango.FontDescription("normal "))`

and in "def on_spbtn_probe_height_value_changed"
`        gtkspinbutton.modify_font(pango.FontDescription("normal "))`

**Comment by alkabal** [comment]
this is used when you edit value manually

**Comment by kiall** [comment]
Could you submit a PR for this? I'm finding all the code references you've given a little hard to follow :)

**Comment by alkabal** [comment]
Hi


Yes I will try to do this tomorrow !


Thanks for all merge 

i hope you a good evening for Christmas

**Comment by alkabal** [comment]
pr submitted.

br

**Comment by kiall** [comment]
Fixed in https://github.com/linuxcnc-probe-screen/probe-screen-ng/pull/40
